awx/main/tasks.py

```diff
@@ -1478,10 +1479,6 @@ def run(self, pk, **kwargs):
                     module_args = ansible_runner.utils.args2cmdline(
                         params.get('module_args'),
                     )
-                shutil.move(
-                    params.pop('inventory'),
-                    os.path.join(private_data_dir, 'inventory')
-                )
```

Comment:
I want this PR, if nothing else, for this. I hit a lot of errors related to container groups with execution environments that came back to this inventory move... which is something we shouldn't need and shouldn't exist.